packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -370,56 +439,32 @@ def __call__(self, message: str) -> bool:
                         )
 
                         self.logger.info(
-                            "Current startup progress [expected number of node-progress-types=%d]: %s",
+                            "Current startup progress [expected %d types]: %s",
                             len(NodeProgressType.required_types_for_started_service()),
                             f"{json.dumps({k: round(v, 2) for k, v in self._current_progress.items()})}",
                         )
-                self._service_ready = self.got_expected_node_progress_types() and all(
+                return self.got_expected_node_progress_types() and all(
                     round(progress, 1) == 1.0
                     for progress in self._current_progress.values()
                 )
-                return self._service_ready
 
         _current_timestamp = datetime.now(UTC)
-        if _current_timestamp - self._last_poll_timestamp > timedelta(seconds=5):
-            # NOTE: we might have missed some websocket messages, and we check if the service is ready
-            if self.is_service_legacy:
-                url = f"https://{self.get_partial_product_url()}x/{self.node_id}/"
-            else:
-                url = (
-                    f"https://{self.node_id}.services.{self.get_partial_product_url()}"
-                )
-            response = None
-            with contextlib.suppress(
-                PlaywrightTimeoutError, TimeoutError, PlaywrightError
+        if (
+            _current_timestamp - self._last_poll_timestamp
+            > _SERVICE_ROOT_POINT_STATUS_TIMEOUT
+        ):
+            self._last_poll_timestamp = datetime.now(UTC)
+            if _check_service_endpoint(
```

Comment:
agreed to change this!

Reply:
so now we will return True in case of a idling of >30 seconds by default.